clusterm panic - invalid memory address or nil pointer dereference

Noticed the following panic. clusterm was left over-night.
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x20 pc=0x4ba606]
goroutine 55 [running]:
github.com/contiv/cluster/management/src/clusterm/manager.(*nodeDiscovered).process(0xc8202ea3e0, 0x0, 0x0)
    /go/src/github.com/contiv/cluster/management/src/clusterm/manager/events.go:47 +0x176
github.com/contiv/cluster/management/src/clusterm/manager.(*Manager).eventLoop(0xc820158190)
    /go/src/github.com/contiv/cluster/management/src/clusterm/manager/events.go:509 +0x15a
    created by github.com/contiv/cluster/management/src/clusterm/manager.(*Manager).Run
    /go/src/github.com/contiv/cluster/management/src/clusterm/manager/manager.go:128 +0x91

```

complete log at https://gist.github.com/vvb/52398d6a36b78c1fe2fa

--- a/management/src/clusterm/manager/events.go
+++ b/management/src/clusterm/manager/events.go
@@ -45,7 +45,7 @@ func (e *nodeDiscovered) process() error {
 	name := e.node.GetLabel() + "-" + e.node.GetSerial()
 
 	enode, err := e.mgr.findNode(name)
-	if err.Error() == nodeNotExistsError(name).Error() {
+	if err != nil && err.Error() == nodeNotExistsError(name).Error() {
 		e.mgr.nodes[name] = &node{
 			// XXX: node's role/group shall come from manager's role assignment logic or
 			// from user configuration

